src/edge_op.cpp

```diff
@@ -206,7 +206,7 @@ void Manifold::Impl::SimplifyTopology() {
   {
     ZoneScopedN("CollapseShortEdge");
     numFlagged = 0;
-    ShortEdge se{halfedge_, vertPos_, tolerance_};
+    ShortEdge se{halfedge_, vertPos_, epsilon_};
```

Comment:
This is my quick fix - seems to be working okay. I need to come back and think about this more deeply sometime, but I think this will be okay for v3.0 - it's pretty close to what we were doing before.

Reply:
Yeah, I think we just need to get the APIs correct for v3.0, detailed implementation can be done a bit later.